src/docs/issue-platform.mdx

```diff
@@ -118,6 +120,21 @@ There's a minimum set of fields which should be included:
 Caveats
 - Attachments will not work. If you have a need for this please contact the issues team.
 
+### Payload Type
+<Link to="https://github.com/getsentry/sentry/blob/ccfb37212102945df5f35e8f55fe38891b70036b/src/sentry/issues/producer.py#L23">PayloadType</Link>
+
+Issue platform supports two types of payloads:
+- `OCCURRENCE`
+- `STATUS_CHANGE`
+
+### Staus Change
+The `status_change` message format is as follows:
+- `fingerprint`: unique identifier for the occurrence
+- `new_status`: status string; one of {`resolved` | `archived` | `unresolved`}
+- `new_substatus` [optional]:
```

Comment:
new_substatus is an int, from the [GroupSubStatus enum](https://github.com/getsentry/sentry/blob/2e5a39ff3dd4afbd33f2ff7d2ba161285107afe4/src/sentry/types/group.py#L4). We do not support the NEW substatus.